api/SVGFECompositeElement.json

```diff
@@ -329,6 +351,54 @@
           }
         }
       },
+      "lighterForError": {
```

Comment:
Robert (domain expert) [confirmed](https://bugzilla.mozilla.org/show_bug.cgi?id=1518099#c13) lighterForError should be added to SVGFECompositeElement too. I have matched the feComposite versions exactly except for Safari. 

For Safari the `operator` was not added until version 6, so I pushed the implementation of operator value lighter to be that same version. That is not "rock solid" (i.e. no bug confirmation) but very likely.

Reply:
Sorry, but we should drop this feature entirely. For reflected attributes, we typically just track whether the attribute exists as part of the API. This is us mostly following the convention of specifications for these things: the IDL usually specifies the reflection, while [the specification for the element defines valid values](https://drafts.fxtf.org/filter-effects/#feCompositeElement).

(Plus—and this is definitely not your fault, since you're following the structure from the element data—the structure is wrong. This ought to be a subfeature of `operator`, not a peer to `operator`.)

Reply:
@ddbeck So to summarize, I just remove this addition altogether - we simply don't track things supported at this level in the BCD. Sounds nuts, but there you go.

Reply:
Yes, we should only have data on the element, not the interface. I got into this more in https://github.com/mdn/browser-compat-data/pull/9074#discussion_r579041812. But let me know if you have more questions—it's not an obvious distinction.

Reply:
@ddbeck What I think you're saying is that we don't document the bits of an SVG element that reflects a javascript API -  we'd just document the javascript behaviour. Right?

So I understand why you wanted me to delete the `lighterForError` from the reflected API  svg/elements/feComposite.json , but THIS is api/SVGFECompositeElement - the API from which we reflect. Which makes me think that it should have stayed. 

What am I missing?

Reply:
> Or did you mean that we document on the element but not the javascript API? Either way, doesn't one of them need comments about the features? The comments here said delete this from both /api and /svg

I meant on the element, not the API. Sorry, I didn't mean to suggest deleting all of them everywhere. I think the various threads on this PR made it hard to keep track of which thing was which. I'll post a standalone review that summarizes things that need to change to get this merged.